src/oscar/apps/search/registries.py

```diff
@@ -0,0 +1,24 @@
+from pydoc import locate
+
+from django.conf import settings
+
+
+class AnalyzerRegistry(object):
+
+    def __init__(self):
+        self.analyzers = {}
+
+    def define_in_index(self, index):
```

Comment:
What problem is this trying to solve? Is this to allow adding analyzers to existing indexes?